courseware/models.py

```diff
@@ -20,3 +21,16 @@ class CoursewareUser(TimestampedModel):
 
     class Meta:
         unique_together = ("user", "platform")
+
+
+class OpenEdXAuth(TimestampedModel):
```

Comment:
Nitpick, take or leave: I'd prefer `OpenEdxAuth` over `OpenEdXAuth`. I know "edX" isn't an acronym, but given the generally-accepted best practice of writing out acronyms as lowercase after the first letter, it feels much more natural to read/write `OpenEdxAuth`. `OpenEdxApiAuth` would be a little more clear about how these objects will be used, but you can take or leave that one as well